src/main/java/frc/robot/autonomous/AutonomousBuilder.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.autonomous;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.commands.PathPlannerAuto;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Robot;
+import frc.utils.auto.PathPlannerUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AutonomousBuilder {
+
+	public static Map<String, Command> getAllAutos(Robot robot) {
+		Map<String, Command> autoMap = new HashMap<>();
+		autoMap.put("M231", withResetOdometry(M231(robot)));
+		autoMap.put("Rotate", new PathPlannerAuto("Rotate"));
+		autoMap.put("Rotate 2m", new PathPlannerAuto("Rotate 2m"));
+		autoMap.put("Straight 2m", new PathPlannerAuto("Straight 2m"));
+		return autoMap;
+	}
+
+	private static Command withResetOdometry(PathPlannerAuto auto) {
```

Comment:
maybe make a wrapper to PathPlannerAuto, and when you create it you can add the with reset odometry as 
auto.with... . i think it will be more readable, but not must.

Reply:
I don't want to create a whole class for one function that I'm managing to get by without

Reply:
I think the reset should not be executed by the auto builder. Like if we want to reset pose we will do it outside by getting the starting pose of the auto.
(Now when I write it I thought maybe wrap the co

Reply:
If we move it outside we will need to check if it's an auto or an empty command instead of having the reset inside the command in the chooser

Reply:
How can it be empty command?

Reply:
If we don't choose an auto